src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -34,33 +34,43 @@
                     {{ format(p.versions[0].updated) }}
                 </div>
                 <div class="col-md-1 project-cell"
+                v-tooltip ="tooltipContent('renameProject')"
                     v-if="renameProjectIsEnabled">
-                    <a @click="renameProject($event, p.id)" 
-                    href="" 
-                    v-translate="'renameProject'"></a>
+                    <button class="btn btn-sm btn-red-icons"
+                    @click="renameProject($event, p.id)">
+                    <edit-icon size="20"></edit-icon>
+                    </button>
                 </div>
-                <div class="col-md-1 project-cell">
-                    <a @click="loadVersion($event, p.id, p.versions[0].id)"
-                       href=""
-                       v-translate="'load'"></a>
+
+                <div class="col-md-1 project-cell"
+                v-tooltip ="tooltipContent('load')">
+                    <button class=" btn btn-sm btn-red-icons"
+                    @click="loadVersion($event, p.id, p.versions[0].id)">
+                    <refresh-cw-icon size="20"></refresh-cw-icon>
+                    </button>
                 </div>
-                <div class="col-md-1 project-cell">
-                    <a @click="deleteProject($event, p.id)"
-                       href=""
-                       v-translate="'delete'"></a>
+
+                <div class="col-md-1 project-cell" 
+                v-tooltip ="tooltipContent('delete')">
+                    <button class=" btn btn-sm btn-red-icons"
+                    @click= "deleteProject($event, p.id)">
+                    <trash-2-icon size="20"></trash-2-icon>
+                    </button>
                 </div>
-                <div class="col-md-2 project-cell"
-                     v-if="promoteProjectIsEnabled">
-                    <a @click="promoteVersion(
+                
+                <div class="col-md-2 project-cell" v-if="promoteProjectIsEnabled"
```

Comment:
Can we also change this `col-md-2` to `col-md-1` so that the row all fits on one line? Similarly below